examples/cloudml-bee-health-detection/trainer/model.py

```diff
@@ -65,19 +65,16 @@ def create_classifier(config, parameters):
     Returns:
         `tf.estimator.DNNClassifier` with specified features and architecture.
     """
-    # Mean and Standard Deviation Constants for normalization.
-    depth_mean = np.float32(parameters.depth_mean)
-    depth_std = np.float32(parameters.depth_std)
-
     # Columns to be used as features.
-
-    depth = tf.feature_column.numeric_column(
-        'depth',
-        normalizer_fn=(lambda x: (x - depth_mean) / depth_std))
+    subspecies = tf.feature_column.categorical_column_with_vocabulary_list(
+        'subspecies',
+        ['Carniolan','Italian','Russian','Other'])
+    subspecies = tf.feature_column.embedding_column(
```

Comment:
Purely a stylistic question, and I genuinely just want to understand your thinking: It's uncomfortable to me when a variable gets different types assigned to it.  Why did you choose 
```
subspecies = categorical_column
subspecies = embedding_column(subspecies)
```
Instead of 
```
species = embedding_column(categorical_column)
```
?

Reply:
I did it because I don't need these to be in two different variables. But I didn't want to have a huge complicated single line either:

Im applying the second needed transformation but in a different line to make the code more readable.

I could doo all of it at once if you think that is cleaner?

Reply:
Also this is a very common TF pattern
E.G.:
x = input_layer
x = danse_layer(x)
x = danse_layer(x)

I'm doing the same:

column = categotical_column
column  = embedding_layer(column)

Reply:
I like your comparison to the layer-by-layer construction of a model.  I argue with myself back and forth that the difference in typing is the same or different (do I say that `x` is of type `feedforward network` or of type `n-layer feedforward network`, and so does that type change?  Is `subspecies` of type `feature column` or of type `categorical feature column`, etc.)

And regardless, I understand and appreciate your motivation and have no desire to ask you to change it.  I wanted to understand what you were balancing, yourself, and how.  Thank you for the explanations!